index.html

```diff
@@ -19,30 +19,44 @@ <h1>Daniel Eicher</h1>
     </nav>
   </header>
   <figure>
-    <img src="https://source.unsplash.com/random/600x300" alt="Beispielbild">
-    <figcaption>Dies ist ein Beispielbild</figcaption>
+    <img src="https://source.unsplash.com/random/600x300" alt="">
+    <figcaption>This is an example image</figcaption>
   </figure>
+  <section>
+    <h2>Interests</h2>
+    <ul>
+      <li>Family</li>
+      <li>Technology</li>
+      <li>Nature</li>
+    </ul>
+  </section>
+  <section>
+    <h2>Hobbies</h2>
+    <ul>
+      <li>Hiking</li>
+      <li>Coding</li>
+      <li>Reading books</li>
+    </ul>
+  </section>
```

Comment:
Well done that code looks good. But I would recommend you to create a new branch because this is not related to the branch name anymore (img).

Reply:
Very nice ;)Just one thing... I would recommend you to set an id to the `section` tags

Reply:
yes i fixed it, thx.